### ISSUE TITLE
Feature Request: Promotion Code Implementation

### ISSUE THREAD

**Issue by ashishkpaul** [vendure-ecommerce/storefront-qwik-starter#148]
Feature Request: Vendure --> Marketing --> Pormotions --> Coupon code implementation on frontend

![Screenshot_2024-02-07_19-31-23](https://github.com/vendure-ecommerce/storefront-qwik-starter/assets/16308863/beec6401-9e90-427a-a824-381e57a0f266)

![Screenshot_2024-02-07_19-21-57](https://github.com/vendure-ecommerce/storefront-qwik-starter/assets/16308863/44267e3d-1a28-4cc9-8b34-b8d79238e1ce)

**Comment by gioboa** [comment]
@bulecampur is working on that

**Comment by prasmalla** [comment]
@gioboa sowy totally forgot to PR this 😅

**Comment by ashishkpaul** [comment]
@prasmalla any update on this?

**Comment by prasmalla** [comment]
@bulecampur are you still working on this

**Comment by bulecampur** [comment]
@prasmalla No, what I was trying was not working so we paid devs to build it for us.

**Comment by ashishkpaul** [comment]
Is any community member resolve this ?

**Comment by gioboa** [comment]
The great @claudiolor implemented it 👏
